python/tstUtilities.py

```diff
@@ -609,11 +609,44 @@ def getFunctionSignature(api, functionObject):
     return signatureString
 
 
+def isOperator(operatorSuffix):
+    """
+    This will get called with what comes after "operator" or "::operator"
+    in a function name.  We are trying to differentiate between a real opeator
+    like operator++ and a function name that starts with "operator" like
+            operator_to_do_something
+            operator123
+            operators
+    """
+
+    if len(operatorSuffix) > 0:
+        firstCharacter = operatorSuffix[0]
+        if (
+            firstCharacter.isalpha()
+            or firstCharacter.isdigit()
+            or firstCharacter == "_"
+        ):
+            return False
+        else:
+            return True
+    else:
+        return False
+
+
 def functionIsOperator(functionName):
     """
     So that we have one place to adjust if we find bugs :)
     """
-    return "::operator" in functionName or functionName.startswith("operator")
+    returnValue = False
+    if "::operator" in functionName:
+        operatorIndex = functionName.find("::operator")
+        returnValue = isOperator(functionName[operatorIndex + len("::operator") :])
```

Comment:
Like this can be:

`returnValue = isOperator(functionName.split("::operator") + "::operator")`

which avoids the explicit find + list splice.